docs/src/main/sphinx/sql/create-table-as.md

```diff
@@ -3,7 +3,7 @@
 ## Synopsis
 
 ```text
-CREATE TABLE [ IF NOT EXISTS ] table_name [ ( column_alias, ... ) ]
+CREATE [ OR REPLACE ] TABLE [ IF NOT EXISTS ] table_name [ ( column_alias, ... ) ]
```

Comment:
Add something about "OR REPLACE" in the Description .. at least that it depends on connector support and what it does to some degree (replace exiting table of same name or whatever..)